Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,24 @@
 #include "BatteryStateOfChargeService.h"
+#include <QDebug>
+#include <QFile>
+#include <QString>
+#include <QTextStream>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    const double MS_TO_HOURS = 3600000.0;
```

Comment:
Think about consistency, we have the abbreviations of `MS`, but use the full word for `HOURS`, in general, there's no harm in using a larger variable name, especially if it represents a constant.